.github/release.yml

```diff
@@ -0,0 +1,11 @@
+changelog:
+  exclude:
+    labels:
+      - not-in-changelog
+      - fix-agda-whitespace
+      - travis
+      - typo
+      - Makefile
+      - release
```

Comment:
Should this list be the same as the CIFM one: https://github.com/agda/agda/blob/1c449e23b1db2462e8bd6959f423898a8fdc361f/src/release-tools/closed-issues-for-milestone/Main.hs#L115-L146
If yes, maybe the latter should be read from this YAML file to avoid duplication.

Reply:
You're right -- I will revise it.